mysql/config-gen

```diff
@@ -16,7 +17,7 @@ echo export VMAIL_DB_BIND_PASSWD=TEMP_VMAIL_DB_BIND_PASSWD
 echo export VMAIL_DB_ADMIN_PASSWD=TEMP_VMAIL_DB_ADMIN_PASSWD
 echo export MYSQL_ROOT_USER='root'
 echo export MYSQL_ROOT_PASSWD="$(openssl rand -hex 32)"
-echo export DEFAULT_PASSWORD_SCHEME='SSHA'
+echo export DEFAULT_PASSWORD_SCHEME="SSHA512"
```

Comment:
This does not seem fine. Didnt you want to put $PASSWORD_SCHEME here?

Reply:
Sorry i will change